Controllers/AuthenticationController.cs

```diff
@@ -17,10 +17,10 @@ public AuthenticationController(IAuthenticationService authenticationService)
     }
 
     [AllowAnonymous]
-    [HttpPost("isEmailAvailable")]
-    public async Task<bool> isEmailAvailable([FromBody] isEmailAvailableRequest request)
+    [HttpPost("isEmailExists")]
+    public async Task<bool> isEmailExists([FromBody] isEmailExistsRequest request)
```

Comment:
IsemailExist has no meaning in english. Please rename it , EmailExists is good but what is isemailexists?

Reply:
Sorguladigimiz icin yazdım o şekilde ama "exists" in fiil olmasina dikkat etmemiştim. isEmailExisting nasil olur?